apps/lib/apps.c

```diff
@@ -847,6 +906,111 @@ int load_crls(const char *file, STACK_OF(X509_CRL) **crls, int format,
     return load_certs_crls(file, format, pass, desc, NULL, crls);
 }
 
+/*
+ * Load those types of credentials for which the result pointer is not NULL.
+ * For each type the first credential found in the store is loaded.
+ * May yield partial result even if rv == 0.
+ */
+int STORE_load_key_cert_crl(const char *uri, const char *pass, const char *desc,
+                            EVP_PKEY **ppkey, X509 **pcert, X509_CRL **pcrl)
```

Comment:
I don't see why you're making duplicates of the already existing `load_` functions.  They aren't precious in an absolute stability manner, libapps is an *internal* library.

Reply:
Good point, so I'll merge them, and tend to add/"export" just 
```
load_key_cert_crl(const char *uri, const char *pass, const char *desc,
                            EVP_PKEY **ppkey, X509 **pcert, X509_CRL **pcrl)
```
for its flexibility loading multiple credentials from the same PEM or PKCS#12 file.

Reply:
This is also done in #11755.